docs/UserGuide.md

```diff
@@ -384,56 +442,133 @@ Format: `c-clear`
+Archives the specified employee from tCheck.
 
 Format: `c-archive INDEX`
 
-* Archives the employee at the specified `INDEX`.
-* The index refers to the index number shown in the displayed `employees' contact details` panel.
-* The index **must be a positive integer** 1, 2, 3, ...​
+* Archives the employee at the specified `INDEX` in the _Employee Directory_ pane.<br>
+* The `INDEX` refers to the index number shown in the current _Employee Directory_ pane.<br>
+* The `INDEX` **must be a positive integer** (e.g. 1, 2, 3, ...​).<br>
+* The `INDEX` **should not be larger than** the total number of employees listed in the current 
+_Employee Directory_ pane. (e.g. There are 2 employees listed in the current _Employee Directory_ pane. The `INDEX`
+should be less or equal to 2.)<br>
 
-<div markdown="span" class="alert alert-primary">:bulb: 
+**:information_source: Notes:**<br>
+
+* If _Employee Directory_ currently shows a list of archived employees (archived employees are displayed with a red
+ color label, named as `[Archived]`, in front of the name), command `c-active-list` needs to be used first to see a
+  list of all your employees. Then, `c-archive INDEX` can be used to archive a specified employee identified
+  by `INDEX`.<br>
 
-**Tip:**
-Command `c-list` can be used first to show a list of all active employees' contact details. 
-Then, `c-archive INDEX` can be used to archive a specific employee.
-</div>
 
 Examples:
-* `c-list` followed by `c-archive 2` archives the 2nd person in the employees' contact details.
-* `c-find Betsy` followed by `c-archive 1` archives the 1st person in the results of the `find` command.
+* Let's say one of your employees, Alex Yeoh, is no longer working in your bubble tea shop. But he may come back
+to work again in the future. You, as a store manager, want to remove him from your employee directory without
+permanently deleting his information, so that you can retrieve/move back his information to your
+employee directory when he comes back to work again. In this case, you can follow these steps.
+
+Steps to archive the first employee, Alex Yeoh, in the sample data:
+1. Type `c-archive 1` into the _Command Box_.
+2. Press `Enter` to execute.
+
+
+Outcome:
+1. The _Result Display_ will show a success message with the employee's name that you have archived.
+2. Alex Yeoh's information will be removed from _Employee Directory_ pane.
+
+{Screenshot}
 
 #### 3.11 Archiving all employees : `c-archive-all`
 
-Archives all employees' contact details from the tCheck.
+Archives all employees in the _Employee Directory_ pane.
 
 Format: `c-archive-all`
 
+**:information_source: Notes:**<br>
+
+* If _Employee Directory_ currently shows a list of archived employees (archived employees are displayed with a red
+ color label, named as `[Archived]`, in front of their names), command `c-active-list` needs to be used first to see a
+  list of all your employees. Then, `c-archive-all` can be used to archive all employees.<br>
```

Comment:
Should this be placed in between the div line to show the blue box? Perhaps you could also add something to say "notes" about what? Applies to all similar sections

Reply:
Thanks for pointing out! Will edit accordingly. Thanks